packages/components/src/playground/button.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { css } from "@linaria/core";
+import { styled } from "@linaria/react";
+
+// Temporary props for button to test out css options for variants
+type ButtonProps = React.DetailedHTMLProps<
+  React.ButtonHTMLAttributes<HTMLButtonElement>,
+  HTMLButtonElement
+> & {
+  variant?: "primary" | "secondary";
+};
+
+function Button(props: ButtonProps): JSX.Element {
+  const { variant = "primary", ...rest } = props;
+  const primary = css`
+    --color-background: blue;
+  `;
+  const secondary = css`
+    --color-background: gray;
+  `;
+
+  const ButtonComponent = styled.button`
```

Comment:
this follows the Linaria theming example closely, but i'm not sure it's appropriate for button variants. fine for testing at least